docs/buildah-copy.md

```diff
@@ -49,9 +53,9 @@ buildah copy containerID 'passwd' 'certs.d' /etc
 
 ### `.dockerignore`
 
-If the file .dockerignore exists in the context directory, `buildah copy` reads
-its contents. Buildah uses the content to exclude files and directories from
-the context directory, when copying content into the image.
+If you specify the \fB\fC\-\-ignorefile\fR option to point to a .dockerignore
+path location. Buildah uses the content to exclude files and directories from
+the source directory, when copying content into the image.
```

Comment:
What's the source directory?  How do I know what it is?

Reply:
Would it be the directory specified or the parent directory of the file, if a file is specified in the command line

```
$ buildah copy CID --ignore /tmp/.containerignore /src /dest # /src is context dir
$ buildah copy CID --ignore /tmp/.containerignore /src/file /dest # /src is context dir
```

Reply:
That answer feels a little hand-wavy to me, but it would need to be in the documentation.

Reply:
Decided to not have a default, and require --context dir with buildah COPY/ADD --ignorefile,